src/tests/monitor.test.ts

```diff
@@ -199,3 +193,24 @@ test.serial(
   const triedToReviveServer1 = fs.existsSync('server1');
   t.is(triedToReviveServer1, false);
 });
+
+test.serial('monitor stop should be idempotent', async t => {
+  const context = t.context as ITestContext;
+
+  await context.monitor.stop();
+  await context.monitor.stop();
+
+  t.pass();
```

Comment:
hmm what's this testing here?

Reply:
Just testing double stop doesn't throw...

Reply:
lol okay